registry.json

```diff
@@ -4305,5 +4305,19 @@
     "telegram": "EnvyStakePool",
     "operatorName": "Sean",
     "github": "EtherealStakePool"
+  },
+  {
+    "ticker": "TILI",
+    "poolId": "4fbb9ac6b2c977ee76da892315fb1e26697a87bf8b54917efaa60053",
+    "social": {
+      "website": "https://tilipool.com/",
+      "twitter": "TiliPool",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "TiliPool",
```

Comment:
Please can you change this to the TG handle of an individual user and not a group or channel? Thanks

Reply:
Is there a reason for not allowing a group or channel? Just curious. :)

Reply:
It's so we have an individual to reach out to in case we need to contact the pool